linkage/pvs_like_case_study/generate_simulated_data/generate_simulated_data_small_sample.ipynb

```diff
@@ -0,0 +1,2894 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    # We don't want to throw out events with a missing/invalid date, so we'll fill them with the value *least* likely to be chosen
</pre><p>nit: wrap long lines to make them easier to read</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ihmeuw/vivarium_research_prl/pull/48/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>